.storybook/storyshots-test.js

```diff
@@ -36,13 +36,13 @@ const beforeScreenshot = () =>
 // Allow a small amount of deviation to account for
 // CI running on a different OS than local development.
 const getMatchOptions = () => ({
-  failureThreshold: 0.1,
+  failureThreshold: 0.05,
   failureThresholdType: 'percent',
 });
 
 initStoryshots({
   suite: 'Visual tests',
-  storyNameRegex: /Visual\stest$/i,
+  storyNameRegex: /Visual\stest\s?([a-z]*)?/i,
```

Comment:
Updating this regex to allow us to create more visual tests for components (think variations like single calendar and range calendar) - with the current regex it would only match on visual test and not for example visual test range.

If we named them the same in the stories the snapshot tests complain that a test with the same name has been defined so this losens this up